### PR TITLE
Component/faq

### PR DESCRIPTION
**Pull request by dCzenczek**
I added a new component FAQ. 
In contentful was added 2 models (faq and faqItem) 

Image below shows how its looks for default colors configuration for dark and light theme

image

Meanwhile I saw you added contentful-export.json from dev environment so went back to master, hope it's no problem

### CODE REVIEW

File: components/cms/Faq.vue [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          {{ item.answer }}
+        </v-expansion-panel-content>
+      </v-expansion-panel>
+    </v-expansion-panels>
+  </div>
+</template>
+
+<script>
+
+import CMSMixin from '~/mixins/CMSMixin'
```
Comment (WojtekTheWebDev) [inline components/cms/Faq.vue:29]: Hi @dCzenczek
Thanks for improving this project with your own component :) 
Some review: Are you using this mixin for some reason?

File: contentful-export.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
         "updatedAt": "2021-05-29T18:03:18.122Z",
         "environment": {
           "sys": {
-            "id": "dev",
+            "id": "master",
```
Comment (WojtekTheWebDev) [inline contentful-export.json:18]: @dCzenczek Small favor, can create an export file from your Contentful? I'm not sure if this file will work after changing it manually, so I'll prepare new one after importing your's.

Comment (SikoraWojciech) [reply contentful-export.json:18]: @dCzenczek I created a branch for this changes to check it locally. ;)

**Comment by WojtekTheWebDev** [comment]
@dCzenczek I left you 2 comments and change base branch from master to component/faq to see it locally.

File: components/cms/Faq.vue (hunk 1 above)
Comment (dCzenczek) [reply components/cms/Faq.vue:29]: Im new in vue framework and only after add this import and variable "mixins" the theme value was found in template

File: contentful-export.json (hunk 2 above)
Comment (dCzenczek) [reply contentful-export.json:18]: @SikoraWojciech I exported file from my contentful and manually move changes regarding FAQ component, after changed it I created new project in contentful and than import from this file and everything was working correctly

File: components/cms/Faq.vue (hunk 1 above)
Comment (WojtekTheWebDev) [reply components/cms/Faq.vue:29]: Sorry, I was thinking of another mixin from the project, you use this right :)
More information about vue mixins: https://vuejs.org/v2/guide/mixins.html

**Review by WojtekTheWebDev — APPROVED** [review]
Thanks for contributing. I'm merging it to the component/faq branch to prepare it to merge with master.

**Pull request merged**